Sources/MapboxNavigation/BannerContainerView.swift

```diff
@@ -1,16 +1,20 @@
 import UIKit
 
 // :nodoc:
-@objc(MBBannerContainerView)
-open class BannerContainerView: UIView {
+public class Banner {
     
     // :nodoc:
-    public enum `Type` {
```

Comment:
Removing this public enumeration would normally be a backwards-incompatible change. This class is undocumented, but it has been around for a while. I didn’t spot much usage of BannerContainerView [elsewhere on GitHub](https://github.com/search?q=BannerContainerView+mapboxnavigation&type=code), so I think it should be OK.